README.md

```diff
@@ -1,25 +1,211 @@
 [![codecov](https://codecov.io/gh/chanzuckerberg/cell-census/branch/main/graph/badge.svg?token=byX1pyDlc9)](https://codecov.io/gh/chanzuckerberg/cell-census)
 
-# CELLxGENE Cell Census
+# Cell Census of CZ CELLxGENE Discover
 
-**Status**: Unstable, under rapid development
+**CZ CELLxGENE Discover** ([cellxgene.cziscience.com](https://cellxgene.cziscience.com/)) is a free-to-use data portal hosting a growing corpus of more than **700 single-cell datasets** comprising about **50 million cells** from the major human and mouse tissues. The portal provides a set of visual tools to download and explore the data. **All data is standardized** to include raw counts and a common vocabulary for gene and cell metadata.
 
-This repository contains documentation and example code related to the Chan Zuckerberg CELLxGENE Cell Census, and a client (API) package to simplify accessing the Cell Census data.
+The **Cell Census** provides easy-to-use and efficient computational tooling to access, query, and analyze all RNA data from CZ CELLxGENE Discover. The Cell Census aims to break the barrier of data fragmentation in the single-cell field by presenting a **new access paradigm of cell-based slicing and querying** for all data at CZ CELLxGENE Discover.
```

Comment:
What is the _barrier of data fragmentation_ ? And if there was a long explanation, this could link to an appendix for inquiring minds.

Reply:
I don't understand a few statements:

1. _easy-to_use_ - SOMA is a power user API that requires a time investment.
2. _efficient computational tooling_ - what is meant by _tooling_?

Reply:
I will add some clarifying text to this, it's a fine balance between succinctness and clarity.

Reply:
> efficient computational tooling - what is meant by tooling?

tooling means data and APIs in this introductory paragraph. The work tooling was intentionally used to avoid technical terms -- our audience are wide range of comp bios, many of whom may be unfamiliar with technical concepts like API.

I slowly introduce more technical as the document progresses, but at the beginning I'd like to keep it as digestable as possible for a broad audience

Reply:
> easy-to-use - SOMA is a power user API that requires a time investment

hence some of the UX wrappers for those who want a quick jump in the data